catalog/app/containers/Admin/Buckets.tsx

```diff
@@ -367,21 +455,128 @@ function BucketFields({ name, reindex }: BucketFieldsProps) {
         </M.AccordionSummary>
         <M.Box className={classes.group} pt={1}>
           {!!reindex && (
-            <M.Button variant="outlined" fullWidth onClick={reindex}>
-              Re-index and repair
-            </M.Button>
+            <M.Box pb={2.5}>
```

Comment:
You are already using `classes = useStyles()` in this component. So it's better to place styles there

Reply:
well, this seems a little too trivial and unspecific to make a class for it